tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: add key for postgres repo
+  apt_key:
+    id: ACCC4CF8
+    url: https://www.postgresql.org/media/keys/ACCC4CF8.asc
+    state: present
+
+- name: add postgres repo to apt sources
+  apt_repository:
+    repo: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main"
+    update_cache: yes
```

Comment:
Can we add a `filename parameter`?  "pgdg" is fine.  Just so we know the name of the file we're creating, in case we ever want to automate removing it again.

https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html#parameters